aviary/subsystems/propulsion/utils.py

```diff
@@ -408,10 +407,30 @@ def setup(self):
 
 
 # class InstallationDragFlag(Enum):
-#     '''
+#     """
 #     Define constants that map to supported options for scaling of installation drag.
-#     '''
+#     """
 #     OFF = auto()
 #     DELTA_MAX_NOZZLE_AREA = auto()
 #     MAX_NOZZLE_AREA = auto()
 #     REF_NOZZLE_EXIT_AREA = auto()
+
+
+class PropellerModelVariables(Enum):
+    """
+    Define constants that map to supported variable names in a propeller model.
+    """
+    HELICAL_MACH = 'Helical_Mach'
+    MACH = 'Mach'
+    CP = 'CP'  # power coefficient
+    CT = 'CT'  # thrust coefficient
+    J = 'J'  # advanced ratio
+
+
+default_propeller_units = {
```

Comment:
Why is this being set here as opposed to just listing it in the variable hierarchy? Mach is never going to change units.

Reply:
I don't quite understand your question but this is done to be consistent with how `EngineModelVariables` class sets its units.

Reply:
Ok I see. I thought these variables would all be accessed outside the prop and so would be defined in the variable hierarchy. It's fine ti leave this as it is.